_episodes/02-centralized.md

```diff
@@ -88,128 +150,75 @@ $ git commit
 
 *local*: ![]({{ site.baseurl }}/img/centralized/04-local.svg)
 
-
-### 5. Try to push the change to the upstream repository
+### 5. Push your change as a new branch
 
 ```
-$ git push origin master
+$ git push origin -u yourname/somefeature
 ```
 
-By **"upstream"** we mean here the repository which we have cloned.
-Imagine "upstream" being closer to the main development and your local
-clone to be "downstream".
+Can we leave out the `-u`?
 
+### 6. Submit a pull request
 
-### 6. **Stop here** and discuss why push for most participants was rejected
+Submit a pull request from your branch towards the `master` branch.
+Do this through the web interface.
 
-You probably see something like this:
+A pull-request means: "please review my changes and if you agree, merge them with a mouse-click".
```

Comment:
```suggestion
A pull-request means: "please review my changes and if you agree, merge them with a mouse-click". In a popular project, it means that anyone can
contribute with *almost no work* on the maintainer's side - a big win.
```
This is an opinion that maybe doesn't need to be here (I'm not sure if it's somewhere else already), but at some point it's worth pointing out that it makes contributions easy - unlike ages ago.